app/services/stats/compute_stats.rb

```diff
@@ -276,21 +269,38 @@ def applicants_for_30_days_rdvs_seen_scope
     # -----------------------------------------------------------------------------------------
     # ----------------------------- Rate of rdvs taken in autonomy ----------------------------
 
-    def rate_of_rdvs_created_by_user
-      compute_rate_of_rdvs_taken_in_autonomy(relevant_rdvs)
+    def rate_of_applicants_autonomy
+      compute_rate_of_applicants_autonomy(relevant_invited_applicants)
     end
 
-    def rate_of_rdvs_created_by_user_grouped_by_month
-      rate_of_rdvs_created_by_user_by_month = {}
-      relevant_rdvs_by_month.each do |date, rdvs|
-        result = compute_rate_of_rdvs_taken_in_autonomy(rdvs)
-        rate_of_rdvs_created_by_user_by_month[date] = result.round
+    def rate_of_applicants_autonomy_grouped_by_month
+      rate_of_applicants_autonomy_by_month = {}
+      relevant_invited_applicants_by_month.each do |date, applicants|
+        result = compute_rate_of_applicants_autonomy(applicants)
+        rate_of_applicants_autonomy_by_month[date] = result.round
       end
-      rate_of_rdvs_created_by_user_by_month
+      rate_of_applicants_autonomy_by_month
+    end
+
+    def compute_rate_of_applicants_autonomy(selected_applicants)
+      relevant_rdvs_created_by_user = relevant_rdvs.select(&:created_by_user?)
+      ids_of_relevant_applicants_with_rdv_created_by_user =
+        selected_applicants.map(&:id) & relevant_rdvs_created_by_user.flat_map(&:applicant_ids)
+      (relevant_invited_applicants.where(id: ids_of_relevant_applicants_with_rdv_created_by_user).count / (
+        selected_applicants.count.nonzero? || 1
```

Comment:
Il faut diviser par les `selected_applicants` qui ont été invité non ? 
Peut-être à prendre en compte par rapport à ma remarque au dessus

Reply:
Les `selected_applicants` sont déjà des utilisateurs invités : selon les cas, ce sont soit des `relevant_invited_applicants` ou des "tranches" de `relevant_invited_applicants_by_month`

Reply:
ah donc le check ` applicant.id.in?(relevant_invited_applicants.ids)` est en trop non ?

Reply:
exact

Reply:
je l'ai enlevé